src/js/apps/patients/schedule/reduced_schedule_app.js

```diff
@@ -38,16 +40,23 @@ export default App.extend({
   },
   onStart(options, collection) {
     this.collection = collection;
+    this.filteredCollection = collection.clone();
 
     this.showScheduleList();
     this.showSearchView();
+    this.showCountView();
   },
   showScheduleList() {
     const collectionView = new ScheduleListView({
       collection: this.collection.groupByDate(),
       state: this.getState(),
     });
 
+    this.listenTo(collectionView, 'filtered', filtered => {
+      this.filteredCollection.reset(filtered);
+      this.showCountView();
+    });
```

Comment:
In order for the count messaging to be updated with the `Find in list...`, I needed to add this `'filtered'` listener to the list and the `this.filteredCollection` that is used in the other schedule and worklist apps.